languages/csharp/docs/README.md

```diff
@@ -1,8 +1,20 @@
 # C# specific docs
 
-## General
-- [How to implement a Concept Exercise](./implementing-a-concept-exercise.md)
+## Reference docs
```

Comment:
I suggest calling this something different to "reference docs" for clarity. Maybe "Student-facing docs"?

Reply:
Excellent!

Reply:
Updated. We should probably rename the "new reference document" GitHub issue template. Agreed? @iHiD